wagtail/core/migrations/0063_collection_management_permissions.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+
+
+def grant_instance_level_collection_management_permissions(apps, schema_editor):
+    Collection = apps.get_model('wagtailcore.Collection')
+    GroupCollectionPermission = apps.get_model('wagtailcore.GroupCollectionPermission')
+    Permission = apps.get_model('auth.Permission')
+
+    # Give all groups who currently manage collections permission to manage root collection
+    root_collection = Collection.objects.filter(depth=1).order_by('path').first()
```

Comment:
If we expect the collection to always exist here, we should use `get()` - If not, we should handle the `root_collection is None` case and gracefully bow out here :)

Reply:
The way I changed the migrations, they just do nothing if there are no root collections.

The case I had had in mind when I wrote that line was a problem someone had in the support channel in slack - where they had more than one root collection in their app. With the changes in our collection management views, I think everything should work even if an app somehow ends up with more than one 'root' collection (in TreeBeard's modeling, that is any collection with depth = 1) so I altered the migrations so they will run no matter how many root collections an app already has - even though the expectation is still that each wagtail installation will have a single root collection.